app/models/photo.rb

```diff
@@ -0,0 +1,11 @@
+class Photo < ActiveRecord::Base
+
+  belongs_to :user
+  has_many :tags
+
+  validates :description, length: { maximum: 200 }
+
+# so that most recent photos are always on top when displayed
+  default_scope { order(:id => :desc) }
```

Comment:
I'd use a named scope instead of a default scope. Then use that scope in your controller method.